packages/jellyfish-api-core/__tests__/category/account.test.ts

```diff
@@ -287,7 +292,7 @@ describe('masternode', () => {
     it('should getTokenBalances with pagination limit', async () => {
       await waitForExpect(async () => {
         const tokenBalances = await client.account.getTokenBalances()
-        expect(tokenBalances.length).toBe(2)
+        expect(tokenBalances.length).toBe(3)
```

Comment:
Increase this to fix failed test. The number increase probably due to adding line 48